jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -494,15 +494,15 @@ public void addExports(Module fromModule, String pkg) {
 /*[ENDIF] JAVA_SPEC_VERSION >= 16 */
 
 /*[IF JAVA_SPEC_VERSION >= 17]*/
-	/*[IF (JAVA_SPEC_VERSION >= 25) & !INLINE-TYPES]*/
+	/*[IF (JAVA_SPEC_VERSION == 25) & !INLINE-TYPES]*/
 	public int uncheckedDecodeASCII(byte[] srcBytes, int srcPos, char[] dstChars, int dstPos, int length) {
 	/*[ELSE] (JAVA_SPEC_VERSION >= 25) & !INLINE-TYPES */
```

Comment:
Please update the comments that repeat the modified conditions.

Reply:
Updated